zerver/lib/actions.py

```diff
@@ -7563,14 +7565,16 @@ def do_revoke_user_invite(prereg_user: PreregistrationUser) -> None:
     Confirmation.objects.filter(content_type=content_type, object_id=prereg_user.id).delete()
     prereg_user.delete()
     clear_scheduled_invitation_emails(email)
-    notify_invites_changed(prereg_user)
+    notify_invites_changed(realm)
 
 
 def do_revoke_multi_use_invite(multiuse_invite: MultiuseInvite) -> None:
+    realm = multiuse_invite.referred_by.realm
+
     content_type = ContentType.objects.get_for_model(MultiuseInvite)
     Confirmation.objects.filter(content_type=content_type, object_id=multiuse_invite.id).delete()
     multiuse_invite.delete()
-    notify_invites_changed(multiuse_invite.referred_by)
+    notify_invites_changed(realm)
```

Comment:
Here we were doing `multiuse_invite.referred_by` on a `multiuse_invite` that has been `.delete()`d. Again, it happens to work, but we don't want to do that.